parts/linux/cloud-init/artifacts/kubelet.service

```diff
@@ -2,10 +2,11 @@
 Description=Kubelet
 ConditionPathExists=/usr/local/bin/kubelet
 Wants=network-online.target
-After=network-online.target
+After=network-online.target containerd.service
 
 [Service]
 Restart=always
+RestartSec=2
 EnvironmentFile=/etc/default/kubelet
 SuccessExitStatus=143
```

Comment:
unrelated but can you figure out what 143 is and leave a comment

not blocking